notebooks/community/model_garden/model_garden_pytorch_instructpix2pix.ipynb

```diff
@@ -0,0 +1,454 @@
+{
```

Comment:
<p>I suggest explaining the model is inside the container</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1637/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
I used a comment below to explain:
`# The pre-built serving docker image. It contains serving scripts and models.`